src/Components/TransactionsList/TransactionsList.js

```diff
@@ -88,10 +89,16 @@ const transactionTableConf = [
         renderColumn: tx => <NetworkColumn network={tx.network}/>,
     },
     {
-        label: 'ETH Value',
+        label: 'Ether Value',
         inclusionKey: TransactionsListColumnTypes.VALUE,
-        size: 120,
-        renderColumn: tx => <div className="TextAlignRight">{parseEthValue(tx.value)} ETH</div>,
+        size: 160,
+        renderColumn: tx => {
+            return (
+                <>
+                    {parseInt(tx.value) ? (<ETHValueLabel value={tx.value} id={tx.id}/>) : (<span>-</span>)}
+                </>
+            )
+        }
```

Comment:
More of a code style comment, this could be written in the following way

```suggestion
        renderColumn: tx => parseInt(tx.value) ? <ETHValueLabel value={tx.value} id={tx.id}/> : <span>-</span>,
```

Also maybe add casting into boolean to make sure that parseInt will return false on 0 value `!!parseInt(tx.value)`